rusk-abi/src/module/host.rs

```diff
@@ -10,6 +10,7 @@ use alloc::vec::Vec;
 use canonical::{Canon, CanonError, Source};
 use dusk_abi::{ContractId, HostModule, Query, ReturnValue};
 use dusk_bls12_381::BlsScalar;
+use dusk_bls12_381_sign::{Signature as BlsSignature, APK as BlsPublicKey};
```

Comment:
I would avoid to alias APK as PublicKey, it can lead to misunderstanding IMO

Reply:
Renamed to `AggregatedBlsPublicKey`